packages/transitive-overlay/src/index.tsx

```diff
@@ -126,8 +179,34 @@ const TransitiveCanvasOverlay = ({
     zoomToGeoJSON(polyline.toGeoJSON(activeLeg.legGeometry.points));
   }, [activeLeg]);
 
+  // Text/symbol layers placed first will be rendered last.
   return (
     <Source data={geojson} id="itinerary" type="geojson">
+      <Layer
+        filter={["==", "type", "from-to"]}
+        id="from-to-labels"
+        layout={defaultTextLayoutParams}
+        paint={defaultTextPaintParams}
+        type="symbol"
+      />
+      <Layer
+        filter={accessLegFilter}
+        id="access-leg-circles"
+        paint={{
+          "circle-color": ["get", "color"],
+          "circle-radius": 6,
```

Comment:
Any way we can make these blobs a bit bigger? To match `itinerary-body`

Reply:
Is this comment geared towards the circles that denote transit stops?

Reply:
Yes! The bikeshare/scootershare ones too

Reply:
Handled in https://github.com/opentripplanner/otp-ui/pull/452/commits/dcd638db37dca2e2cf56f699818b6f2afb9fc37a.